test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3659,6 +3671,28 @@ def forward(self, input):
         model = MyModule()
         self.run_test(model, (x,))
 
+    def test_dtype(self):
```

Comment:
Can you please locally verify that all tests pass with 'TestONNXRuntime_opset12_IRv4_new_jit_API' ?